includes/Story_Post_Type.php

```diff
@@ -574,6 +604,31 @@ public function get_editor_settings() {
 		return apply_filters( 'web_stories_editor_settings', $settings );
 	}
 
+	/**
+	 * Setup up post lock.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param int $story_id Post id of story.
+	 *
+	 * @return void
+	 */
+	protected function setup_lock( $story_id ) {
+		if ( current_user_can( 'edit_post', $story_id ) ) {
```

Comment:
This should probably check `$post_type_object->cap->edit_post` instead

Reply:
Fixed.